### ISSUE TITLE
Feature: JetBrains Runtime variant for Java11 

### ISSUE THREAD

**Issue by martin-jamszolik** [sdkman/sdkman-db-migrations#465]
**Feature request**
JetBrains is maintaining an excellent variant of Java11 and Java8 with fixes for desktop and GUI application to help with Subpixel Anti-Aliasing, enhanced font rendering on Linux, HiDPI support, ligatures among few other fixes.

https://confluence.jetbrains.com/display/JBR/JetBrains+Runtime

Did this project ever consider listing JBR as an option?
Thank you!

**Comment by marc0der** [comment]
@eddumelendez does the Disco API have any knowledge of this JDK distribution? If so, maybe we could automate this.

If not, it could always be added with a PR on this repo @martin-jamszolik. Please see the README for more details.

**Comment by eddumelendez** [comment]
I don't think it is provided by Disco API because this one is not certified. not sure how often people use it. From the link shared we can read

> JetBrains Runtime is not a certified build of OpenJDK. Please, use at your own risk.

**Comment by helpermethod** [comment]
@eddumelendez It seems it has been added to DiscoAPI in the meanwhile.
